modules/configuring/proc-provisioning-your-custom-configuration.adoc

```diff
@@ -20,8 +20,13 @@ It contains one secret per line in `KEY=value` form.
 
 . Author your custom `{my-app-config-file}` file.
 This is the main {product-short} configuration file.
```

Comment:
```suggestion
This is the main {product-short} configuration file.
You need a custom `{my-app-config-file}` file to avoid the {product-short} installer to revert user edits during upgrades.
When your custom `{my-app-config-file}` file is empty, {product-short} is using default values.
```